website/content/docs/connect/proxies/envoy-extensions/usage/lua.mdx

```diff
@@ -15,11 +15,9 @@ Envoy filters support setting and getting dynamic metadata, allowing a filter to
 
 To use the Lua Envoy extension, configure the following arguments in the `EnvoyExtensions` block:
 
-| Arguments      | Description                                                                                      |
```

Comment:
Do we need the header from this table still?

Reply:
I changed it from a table to a list so that it is more consistent with the configuration docs for the other extensions. Because there are only 3 config fields, I didn't think that it was worth creating a new config section for the Lua filter. Do you think this is OK?